hack/boilerplate/boilerplate.py

```diff
@@ -151,21 +147,16 @@ def file_extension(filename):
     return os.path.splitext(filename)[1].split(".")[-1].lower()
 
 
-skipped_dirs = ['third_party', '_gopath', '_output', '.git', 'cluster/env.sh',
-                "vendor", "test/e2e/generated/bindata.go", "hack/boilerplate/test",
-                "staging/src/k8s.io/kubectl/pkg/generated/bindata.go"]
-
-# list all the files contain 'DO NOT EDIT', but are not generated
-skipped_ungenerated_files = [
-    'hack/update-generated-swagger-docs.sh',
-    'hack/boilerplate/boilerplate.py'
-]
+skipped_names = ['hack', 'third_party', '_gopath', '_output', '.git',
```

Comment:
FWIW, we do put boilerplate at the top of hack scripts, this will stop verifying/requiring it. Not sure if that's ok or not, just noting it.

Reply:
Ah, yes, of course.  And this is why we had a specific exclusion for update-generated-swagger-docs.sh, because it has the magic string in it.

I can fix that, I think.  This is all painfully dumb :)